Sources/MapboxMaps/Gestures/GestureOptions.swift

```diff
@@ -73,5 +73,8 @@ public struct GestureOptions: Equatable {
     /// Defaults to `UIScrollView.DecelerationRate.normal.rawValue`
     public var panDecelerationFactor: CGFloat = UIScrollView.DecelerationRate.normal.rawValue
 
+    /// Whether touching with the map cancels and prevents animations
+    public var animationLockoutEnabled: Bool = true
+
```

Comment:
Need to evaluate whether this is the right solution. This was just added quickly (and without tests) for demonstration purposes.

Reply:
Removed from this PR for now. Will revisit in a future PR.